### PR TITLE
tests: Add docker search test

### LINKED ISSUE #448: add docker search test
add integration docker search test

### PR DESCRIPTION
**Pull request by GabyCT**
Add integration docker search test.

Fixes #448

Signed-off-by: Gabriela Cervantes Tellez <gabriela.cervantes.tellez@intel.com>

### CODE REVIEW

File: integration/docker/search_test.go [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+
+package docker
+
+import (
+	. "github.com/clearcontainers/tests"
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+)
+
+var _ = Describe("search", func() {
```
Comment (devimc) [inline integration/docker/search_test.go]: I think this should be ```"docker search"```

File: integration/docker/search_test.go [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+	var (
+		args []string
+	)
+
+	Context("search an image", func() {
+		It("should filter the requests", func() {
+			args = []string{"--filter", "is-official=true", "--filter=stars=3", Image}
```
Comment (devimc) [inline integration/docker/search_test.go]: I guess currently stars is 3 but what will happen if tomorrow is 4 o 2? does this test fail if the stars of the image change?

**Comment by GabyCT** [comment]
@devimc changes applied

File: integration/docker/search_test.go (hunk 2 above)
Comment (GabyCT) [inline integration/docker/search_test.go]: @devimc the starts 3 means at least 3 stars so it includes from 3 to more

**Pull request merged**